tests/test_sourceTable.py

```diff
@@ -258,6 +258,10 @@ def testColumnView(self):
         self.assertFloatsEqual(cols2["c_xx"], cols2.getIxx())
         self.assertFloatsEqual(cols2["c_yy"], cols2.getIyy())
         self.assertFloatsEqual(cols2["c_xy"], cols2.getIxy())
+        self.catalog.table.schema.getAliasMap().erase("slot_Centroid")
+        self.catalog.table.schema.getAliasMap().erase("slot_Shape")
+        for quantity in ["X", "Y", "Ixx", "Iyy", "Ixy"]:
+            self.assertRaises(lsst.pex.exceptions.LogicError, getattr(self.catalog, f"get{quantity}"))
```

Comment:
Again with the `with`. Please add a comment above this explaining what this test is doing: it took me a bit to figure it out.